src/main/webapp/resources/js/components/samples/components/SampleFiles.tsx

```diff
@@ -31,22 +40,22 @@ import { SPACE_MD } from "../../../styles/spacing";
  */
 export function SampleFiles() {
   const { sample, projectId, modifiable } = useSelector(
-    (state) => state.sampleReducer
+    (state: RootStateOrAny) => state.sampleReducer
   );
   const { files, loading, concatenateSelected } = useSelector(
-    (state) => state.sampleFilesReducer
+    (state: RootStateOrAny) => state.sampleFilesReducer
```

Comment:
Update

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f